### PR TITLE
feat: always use latest expo and update packages

### PR DESCRIPTION
**Pull request by dannyhw**
## Description

Installs expo@latest and then runs expo install --fix immediately after to make sure we have the latest compatible versions of everything. 

## Motivation and Context

This reduces the need to manually update package versions.

## Screenshots (if appropriate):

image

### CODE REVIEW

File: cli/src/utilities/printOutput.ts [hunk 1]
```diff
@@ -34,14 +34,32 @@ export async function printOutput(
   const packageManager = cliResults.flags.packageManager || getPackageManager(toolbox, cliResults);
 
   if (!options.noInstall && !flags.noInstall) {
-    s.start(`Installing dependencies using ${packageManager}...`);
+    s.start(`Installing dependencies using ${packageManager}... `);
```
Comment (danstepanov) [inline cli/src/utilities/printOutput.ts:37]: why the extra space here?

File: cli/src/utilities/printOutput.ts [hunk 2]
```diff
@@ -34,14 +34,32 @@ export async function printOutput(
...
+    s.stop('latest version of Expo installed!');
+
+    s.start('updating packages to expo compatible versions...');
+
+    await system.spawn(`cd ${projectName} && ${packageManager} expo install --fix`, {
+      shell: true,
+      stdio: ['ignore', 'ignore', 'inherit']
+    });
+
+    s.stop('packages updated!');
```
Comment (danstepanov) [inline cli/src/utilities/printOutput.ts:61]:
```suggestion
    s.stop('Packages updated!');
```

File: cli/src/utilities/printOutput.ts [hunk 3]
```diff
@@ -34,14 +34,32 @@ export async function printOutput(
...
+    s.start('Updating Expo to latest version...');
+
+    await system.spawn(`cd ${projectName} && ${packageManager} install --silent expo@latest`, {
+      shell: true,
+      stdio: ['ignore', 'ignore', 'inherit']
+    });
+
+    s.stop('latest version of Expo installed!');
+
+    s.start('updating packages to expo compatible versions...');
```
Comment (danstepanov) [inline cli/src/utilities/printOutput.ts:54]:
```suggestion
    s.start('Updating packages to expo compatible versions...');
```

File: cli/src/utilities/printOutput.ts [hunk 4]
```diff
@@ -34,14 +34,32 @@ export async function printOutput(
...
     s.stop('Dependencies installed!');
 
+    s.start('Updating Expo to latest version...');
+
+    await system.spawn(`cd ${projectName} && ${packageManager} install --silent expo@latest`, {
+      shell: true,
+      stdio: ['ignore', 'ignore', 'inherit']
+    });
+
+    s.stop('latest version of Expo installed!');
```
Comment (danstepanov) [inline cli/src/utilities/printOutput.ts:52]:
```suggestion
    s.stop('Latest version of Expo installed!');
```

File: cli/src/utilities/printOutput.ts (hunk 1 above)
Comment (dannyhw) [reply cli/src/utilities/printOutput.ts]: a mistake, removed it

**Comment by dannyhw** [comment]
ok @danstepanov added a changeset, does that look right?

**Review by danstepanov — APPROVED** [review]
LGTM

**Pull request merged**